lms/static/sass/views/_homepage.scss

```diff
@@ -49,7 +50,7 @@ $learn-more-horizontal-position: calc(50% - 100px); // calculate the left positi
 
         img {
           width: 100%;
-          height: auto;
+          min-height: $course-image-height;
```

Comment:
This should I think revert back to auto to account for responsive sizing needs. Hopefully this isn't critical to passing a11y tests.